### DISCUSSION TITLE
21-06-23 일일회고

### DISCUSSION THREAD

**Discussion by eeeesong** [d-h-k/issue-tracker#102] [회고]
오늘도 한번 제가 올려보겠습니다!

**Comment by eeeesong** [comment]
오늘은 이슈 작성 화면에서 레이블, 마일스톤, 담당자를 선택하는 모달 창들을 만들고
새 이슈 POST API를 적용해보았습니다!

**Comment by JacksonPk** [comment]
저는 이슈메인화면창을 만들었고 내일까지 디테일로 해당 이슈에 코멘트를 다는것을 해보려구 합니다!!
다들 화이팅입니다!

**Comment by Jiwon-JJW** [comment]
이슈 관련에 대해서 말씀해 주신 것이 있어서 수정 하고난 후 검색 필터를 만들어보고 있었습니다!!
